app/views/users/edit.html.erb

```diff
@@ -5,4 +5,25 @@
 <script id="user-edit__props" type="application/json">
   <%= presenter.props_to_json.html_safe %>
 </script>
+<div class="container mx-auto px-3 py-8 max-w-5xl">
+  <div class="bg-white shadow sm:rounded-lg">
+    <div class="px-4 py-5 sm:p-6">
+      <div class="flex flex-col md:flex-row">
+        <div class="w-full md:w-1/3 pr-4">
+          <h3 class="text-lg font-semibold">Connect you Discord Account</h3>
+          <p class="mt-1 text-sm text-gray-600">This will help you gain access to locked channels in discord</p>
```

Comment:
1. Move strings to `en.yml`.
2. Always capitalize the first letter of `Discord`.